sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/Docs/ConversationAnalysisClient.xml

```diff
@@ -461,7 +461,7 @@ var data = new {
     },
 };
 
-var operation = await client.StartAnalyzeConversationAsync(WaitUntil.Completed, RequestContent.Create(data));
```

Comment:
Has the recommendation changed? We were previously told to use `WaitUntil.Completed` in samples. If anything, I'd expect that the call to `WaitForCompletionAsync()` should be removed and just get the `operation.Value`.

Our hand-written and generated samples should probably align. /cc @m-nash @pallavit @jsquire

Reply:
@heaths The change is to fix a problem in our generated sample codes: we pass in `WaitUntil.Completed` then we should not invoke `WaitForCompletionAsync()`. So I just change the passed-in parameter to `WaitUntil.Started`, without being aware of the recommendation.
I'll make another change to follow the recommendation. Thanks.

Reply:
Fixed in https://github.com/Azure/autorest.csharp/issues/2984